src/style.css

```diff
@@ -1,3 +1,74 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
According to the checklist, you should use fallback fonts. In case the 'Roboto' font doesn't load, the browser will use the fallback fonts. You've used 'Arial' and 'sans-serif' as fallbacks which is a good practice, but it's better to add more options. For example: `font-family: Roboto, Arial, Helvetica, sans-serif;`